bootstrapvz/plugins/file_copy/README.rst

```diff
@@ -0,0 +1,26 @@
+File copy
+---------
+
+This plugin lets you copy files from the host to the VM under construction,
+  create directories, and set permissions and ownership.
+
+Note that this necessarily violates the `first development guideline`_.
```

Comment:
Good catch! It does.  
Though this plugin is more for users than developers. e.g. I'd never accept PRs for example/official manifests that make use of this plugin. But it's a great way for people to pre-install a lot of stuff on their custom images (the same could also be said for the `command` and the various config management plugins).